### ISSUE TITLE
Recent log messages - small visual improvement.

### ISSUE THREAD

**Issue by findlabnet** [backdrop/backdrop-issues#1102]
Messages of type _"Login attempt failed for %user"_ (in my opinion) need to be marked as warning action.
Just severity parameter should be added to call watchdog, line 1601 in **user.module**:

``` php
watchdog('user', 'Login attempt failed for %user.', array('%user' => $form_state['values']['name']), WATCHDOG_WARNING); 
```

Not for immediately PR, but can be added with any commit to the module.

**Comment by findlabnet** [comment]
After core's 1.2.x improvement, this fix can be issued twice:
on line 1604
```php
watchdog('user', 'The user account %name could not be found.', array('%name' => $form_state['values']['name']), WATCHDOG_WARNING);
```
and on line 1608
```php
watchdog('user', 'Login attempt failed for %user.', array('%user' => $form_state['values']['name']), WATCHDOG_WARNING);
```

**Comment by quicksketch** [comment]
Sounds good to me.

**Comment by findlabnet** [comment]
BTW, why we help to bruteforcer understand whats wrong in this attempt: username or password?

**Comment by quicksketch** [comment]
> BTW, why we help to bruteforcer understand whats wrong in this attempt: username or password?

This article outlines a lot of really great information on login/password usability: http://blog.mailchimp.com/social-login-buttons-arent-worth-it/

In particular, this bit applies directly to Backdrop:

> The engineering team, ever mindful of security, argued that being generic about username and password errors makes it harder for bad guys to guess usernames by pounding the form with random words or email addresses. But after some further consideration, we decided that it was a false risk, as the username reminder form already tells you if a username exists, and is not a significant security risk for the bajilions of sites that have them.

Backdrop does this as well. If you use a password reset form, it already tells you if a username or account is present on a site. So not telling the user if it was either their username or password on the login form is only hurting users and would have no effect on an even moderately clever brute-forcer.

**Comment by findlabnet** [comment]
Thank you for the clarification, Nate!

**Comment by findlabnet** [comment]
Issued [PR 1283](https://github.com/backdrop/backdrop/pull/1283).
Result should be as on top line instead of bottom as now:

![log](https://cloud.githubusercontent.com/assets/11479969/13577083/6b5e58c8-e499-11e5-8237-3ff306e2ca5e.png)

**Comment by quicksketch** [comment]
Hi @findlabnet, it's great to see you made a PR for this issue! I've merged in https://github.com/backdrop/backdrop/pull/1283. Thanks! This is such a minor change, I've included it in the 1.3.x branch for the next bug fix release (1.3.4).

**Comment by findlabnet** [comment]
Just slightly tired to make a patch for myself with each new release. :-)
